.github/workflows/ci.yml

```diff
@@ -74,6 +78,7 @@ jobs:
   test_windows:
     name: Windows Server 2019, Erlang/OTP ${{ matrix.otp_version }}
     strategy:
+      fail-fast: false
```

Comment:
I would prefer to not list this one. The majority of core developers do not develop on Windows, so getting the full run, even in case of failures, is valuable feedback. When it happens on Unix, we can usually reproduce it more easily, so that's not a big concern.

```suggestion
```